apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/FeatureFlagsSettings/index.tsx

```diff
@@ -89,13 +118,36 @@ export const FeatureFlagContent = withV3StyleProvider((props: { visible?: boolea
     [filteredFlags, focusedName],
   );
 
+  const groupsList = useMemo(
+    () =>
+      filteredGroups
+        .sort()
+        .map(groupName => (
+          <GroupedFeatures
+            key={groupName}
+            groupName={groupName}
+            focused={focusedGroupName === groupName}
+            setFocusedGroupName={setFocusedGroupName}
+          />
+        )),
+    [filteredGroups, focusedGroupName],
+  );
+
   const config = useFeature("firebaseEnvironmentReadOnly");
   const params = config?.params;
   const project =
     params !== null && typeof params === "object" && "project" in params
       ? (params as { project: string }).project
       : "";
 
+  const handleChangeTab = useCallback((index: number) => {
+    setActiveTabIndex(index);
+  }, []);
+
+  const setFeatureFlagButtonVisible = useCallback(() => {
+    dispatch(setFeatureFlagsButtonVisible(!featureFlagsButtonVisible));
```

Comment:
This works but could you use the 1st param that the `onChange` function will be called with instead of the state?